ex03.ex

```diff
@@ -77,7 +82,16 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+  def list_contains([ h | t ], a ) do 
```

Comment:
## -2

Again, please don't use `if` for this. One extra deduction for not testing in the pattern match:

~~~ elixir
  def list_contains([], _value),             do: false
  def list_contains([value | _rest], value), do: true
  def list_contains([_ | rest], value),      do: list_contains(rest, value)
~~~